partials/forms/select.html

```diff
@@ -1,12 +1,22 @@
-<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} error{{/error}}">
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} error{{/error}}{{#groupClassName}} {{groupClassName}}{{/groupClassName}}"
+{{#groupAttributes}}
+    {{attribute}}="{{value}}"
+{{/groupAttributes}}
+>
     <label for="{{id}}" class="{{labelClassName}}">
         {{{label}}}
         {{#hint}}<span {{$hintId}}id="{{hintId}}" {{/hintId}}class="form-hint">{{hint}}</span>{{/hint}}
         {{#error}}<span class="error-message">{{error.message}}</span>{{/error}}
     </label>
     <select id="{{id}}" class="{{#className}}{{className}}{{/className}}" name="{{id}}" aria-required="{{required}}">
     {{#options}}
-        <option value="{{value}}" {{#selected}}selected{{/selected}}>{{label}}</option>
+        <option value="{{value}}"
+        {{#selected}}selected{{/selected}}>
+        {{label}}
+        {{#attributes}}
+            {{attribute}}="{{value}}"
+        {{/attributes}}
```

Comment:
Have you tried this? These 'attributes' will render next to the label and not as actual attributes.